slices/libpython3.8-stdlib.yaml

```
@@ -0,0 +1,381 @@
+    contents:
+      /usr/lib/python3.8/netrc.py:
+      /usr/lib/python3.8/plistlib.py:
+      /usr/lib/python3.8/xdrlib.py:
+
+  # File and Directory Access
+  # https://docs.python.org/3.8/library/filesys.html
+  filesys:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/filecmp.py:
+      /usr/lib/python3.8/fileinput.py:
+
+  # Program Frameworks
+  # https://docs.python.org/3.8/library/frameworks.html
+  frameworks:
+    essential:
+      - libpython3.8-stdlib_core
+      - libpython3.8-stdlib_text
+    contents:
+      /usr/lib/python3.8/cmd.py:
+      /usr/lib/python3.8/shlex.py:
+      /usr/lib/python3.8/turtle.py:
+
+  # Importing Modules
+  # https://docs.python.org/3.8/library/modules.html
+  importing:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/modulefinder.py:
+
+  # Internet Protocols and Support
+  # https://docs.python.org/3.8/library/internet.html
+  internet:
+    essential:
+      - libpython3.8-stdlib_core
+      - libpython3.8-stdlib_crypto
+      - libpython3.8-stdlib_file-formats
+      - libpython3.8-stdlib_frameworks
+      - libpython3.8-stdlib_ipc
+      - libpython3.8-stdlib_markup
+      - libpython3.8-stdlib_numeric
+      - libpython3.8-stdlib_pydoc
+      - libuuid1_libs
+    contents:
+      /usr/lib/python3.8/cgi.py:
+      /usr/lib/python3.8/cgitb.py:
+      /usr/lib/python3.8/ftplib.py:
+      /usr/lib/python3.8/http/cookiejar.py:
+      /usr/lib/python3.8/http/cookies.py:
+      /usr/lib/python3.8/http/server.py:
+      /usr/lib/python3.8/imaplib.py:
+      /usr/lib/python3.8/lib-dynload/_uuid.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/nntplib.py:
+      /usr/lib/python3.8/nturl2path.py:
+      /usr/lib/python3.8/poplib.py:
+      /usr/lib/python3.8/smtpd.py:
+      /usr/lib/python3.8/smtplib.py:
+      /usr/lib/python3.8/telnetlib.py:
+      /usr/lib/python3.8/uuid.py:
+      /usr/lib/python3.8/webbrowser.py:
+      /usr/lib/python3.8/wsgiref/**:
+      /usr/lib/python3.8/xmlrpc/**:
+
+  # Networking and Interprocess Communication
+  # https://docs.python.org/3.8/library/ipc.html
+  ipc:
+    essential:
+      - libpython3.8-stdlib_core
+      - libpython3.8-stdlib_concurrency
+    contents:
+      /usr/lib/python3.8/asynchat.py:
+      /usr/lib/python3.8/asyncio/**:
+      /usr/lib/python3.8/asyncore.py:
+      /usr/lib/python3.8/lib-dynload/_asyncio.cpython-38-*-linux-*.so:
+
+  # Python Language Services
+  # https://docs.python.org/3.8/library/language.html
+  language:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/lib-dynload/parser.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/pickletools.py:
+      /usr/lib/python3.8/pyclbr.py:
+      /usr/lib/python3.8/symbol.py:
+      /usr/lib/python3.8/symtable.py:
+      /usr/lib/python3.8/tabnanny.py:
+
+  # Structured Markup Processing Tools (HTML, XML)
+  # https://docs.python.org/3.8/library/markup.html
+  markup:
```

```suggestion
  markup-tools:
```

(to be consistent with "development tools")